internal-packages/replication/src/stream.test.ts

```
@@ -0,0 +1,203 @@
+              payload: JSON.stringify({ foo: "bar" }),
+              traceId: `${i}`,
+              spanId: `${i}`,
+              queue: "test",
+              runtimeEnvironmentId: runtimeEnvironment.id,
+              projectId: project.id,
+            },
+          });
+        }
+
+        resolve(undefined);
+      }).then(() => {});
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)